### ISSUE TITLE
Week starting 4/12

### ISSUE THREAD

**Issue by ddalama** [ddalama/CS253-Presentation#1]
Summary of Friday's meeting with Dr. Grigni:
- Suggested discussing gradient descent and optimization techniques in relation to the k-center problem. 
- could talk about Machine Learning aspects (eg. unsupervised learning) of k-center problem briefly. 

**TODO for this week:** - assign roles to work on presentation. 
- To complete this task, we have to understand the k-center problem well. 
- So, before our meeting, read https://www.cs.umd.edu/class/fall2017/cmsc451-0101/Lects/lect08-greedy-k-center.pdf. Briefly research how machine learning relates to k-center problem and present an ELI5 of what you researched to our group. Think about how we can divide roles and we will divide roles during our meeting on Tuesday.

**Comment by qyccc3** [comment]
here is a resource that I think is very useful
[durham - k-center problems 08.pdf](https://github.com/ddalama/CS253-Presentation/files/6305263/durham.-.k-center.problems.08.pdf)

**Comment by ddalama** [comment]
Roles:
Yongchen: Make slides and work on creative part (eg. code demo)
Dheep: Write script
Matthew: Work on applications of k-centers problem, specifically to ML.

**Comment by ddalama** [comment]
Email IDs:

jackqianhhh@gmail.com
morenomatthew2001@gmail.com
ddalama@emory.edu

**Comment by ddalama** [comment]
Link to Google Slides: https://docs.google.com/presentation/d/1TWJSzb19E8IS8kWDVaEjWGVV7keGGkpPrxGnnNIE3ig/edit?usp=sharing

**Comment by ddalama** [comment]
Link to script 1st draft: 
https://docs.google.com/document/d/18xoWqRHc6QSwB3W0AgdMwBPLj3Rjon5KKyiMhjMW330/edit
